src/components/BusinessesList/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from "react";
+import http from "../../services/httpService";
+import BusinessCard from "../BusinessCard";
+
+const endPointUrl = `https://api.myjson.com/bins/wcdjs`;
+export default function BusinessesList() {
+  const [businesses, setBusinesses] = useState([]);
+
+  useEffect(() => {
+    (async function getBuisinesses() {
```

Comment:
the useEffect takes a function, so no need to wrapper the await in an IIFE, you can add the `async` keyword in the arrow function of the useEffect

Reply:
i read about this issue.
we cant wrap the useEffect with async because it shouldnt return a promise,
thats why i declared async function inside

Reply:
damn i really hate this syntax but oh well